debian/control

```diff
@@ -3,12 +3,12 @@ Section: net
 Priority: extra
 Maintainer: Jitsi Team <dev@jitsi.org>
 Uploaders: Emil Ivov <emcho@jitsi.org>, Damian Minkov <damencho@jitsi.org>, Pawel Domas <pawel.domas@jitsi.org>
-Build-Depends: debhelper, openjdk-8-jre-headless | openjdk-11-jre-headless, maven
+Build-Depends: debhelper, java8-runtime-headless | java8-runtime | java11-runtime-headless | java11-runtime, maven
```

Comment:
Doesn't java8-runtime imply `java8-runtime-headless` (so `java8-runtime-headless | java8-runtime` reduces to `java8-runtime-headless`)

Reply:
AdoptOpenJDK doesn't provide headless variants.